server/functions/ai_objectives/fn_ai_obj_request_pursuit.sqf

```diff
@@ -70,8 +70,26 @@ _objective setVariable ["onTick", {
 	// while other targets nearby are active.
 	private _targetIdx = _targets findIf {!(_x getVariable ["vn_revive_incapacitated", false])};
 
-	// selects the last target in the targets array if everyone is incapacitated
-	// TODO: patrol around or move away?
+	// everyone is incapacitated, let's patrol the area or set up an ambush
+	if (_targetIdx == -1) exitWith {
+
+		private _groups = (_objective getVariable "assignedGroups");
+
+		// only change the group's orders if the orders are currently to pursue,
+		// otherwise we've already changed the orders and don't need to do this
+		if (_groups findIf {((_x getVariable "orders") select 0) isEqualTo "pursue"} > -1) then {
```

Comment:
change this to a `select` --> there may be a scale up of AI at some point, adding new groups with the pursuit objective.

```sqf
// select any group that has a pursuit order
// set a random objective on the group
_groups select {
    // all paradigm AI groups should have orders
    private _ordersString = (_x getVariable "orders") select 0;
    _ordersString isEqualTo "pursue";
} apply {
    // create new orders based on the position of the first player in the targets array
    private _targets = _objective getVariable "targets";
    private _newOrders = selectRandom [
        ["patrol", getPos (_targets select 0), 50],
        ["ambush", getPos (_targets select 0)]
    ];

    _x setVariable ["orders", _newOrders, true];
};
```

```